README.md

```diff
@@ -248,6 +262,33 @@ To use no fallback when token authentication fails, set `fallback: :none`.
 
   [csrf]: https://github.com/gonzalo-bulnes/simple_token_authentication/issues/49
 
+### Testing with Rspec/Minitest
```

Comment:
Also, to avoid confusion, I'd stick to the credentials that were used two paragraphs before for examples:

> X-User-Email alice@example.com
X-User-Token 1G8_s7P-V-4MGojaKD7a

Any thoughts @johnmosesman? : )